### PR TITLE
GCP images maintenance

### PR DESCRIPTION
**Pull request by diasjorge**
I'm implementing operations to cleanup old gcp images to cloud custodian.

Any advise to get this into the project regarding tests, etc would be greatly appreciated.

### CODE REVIEW

**Comment by kapilt** [comment]
Thanks for the pull request!  If you don't mind could you sign the CLA at the bottom of the README? Direct link here: https://docs.google.com/forms/d/e/1FAIpQLSfwtl1s6KmpLhCY6CjiY8nFZshDwf_wrmNYx1ahpsNFXXmHKw/viewform

fwiw, we don't really need a separate age filter here, the default value filter can do age comparisons (its existence in the aws provider is for historical compatiblity).

ie. 

```
type: value
key: creationTimeStamp
value_type: age
value: 120
op: greater-than
```

**Comment by kapilt** [comment]
wrt to tests and contributions on the gcp provider, there's a brand new developer guide
https://github.com/cloud-custodian/cloud-custodian/blob/master/docs/source/gcp/contribute.rst

**Comment by kapilt** [comment]
@diasjorge thanks for adding tests, still need the cla signature before merging.

**Comment by diasjorge** [comment]
@kapilt signed :)

File: tools/c7n_gcp/tests/test_compute.py [hunk 1]
```diff
@@ -138,3 +138,29 @@ def test_snapshot_delete(self):
...
+            'image-delete', project_id='cloud-custodian')
+        p = self.load_policy(
+            {'name': 'all-images',
+             'resource': 'gcp.image',
+             'filters': [
+                 {'name': 'image-1'}],
+             'actions': ['delete']},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```
Comment (kapilt) [inline tools/c7n_gcp/tests/test_compute.py]: fwiw, normally for a test on an action, we would create a client here to get the image again to verify that a state transitioning is happening (ie State != 'READY')

**Pull request merged**